tools/packaging/guest-image/build_image.sh

```diff
@@ -32,7 +35,17 @@ build_initrd() {
 	export USE_DOCKER=1
 	export AGENT_INIT="yes"
 	# ROOTFS_BUILD_DEST is a Make variable
-	sudo -E PATH="$PATH" make rootfs ROOTFS_BUILD_DEST="${rootfs_build_dest}"
+
+	if [ -z "${AA_KBC}" == "offline_sev_kbc" ]; then
```

Comment:
At some point soonish we will want to add support for `online_sev_kbc`. Not for the first release, tho.

Reply:
I'll open an issue regarding the online kbc to make sure I keep track of it.